src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const parentElement = document.querySelector('ul');
+const employees = Array.from(document.querySelectorAll('li'));
+
+function getSalary(employee) {
+  const salaryString = employee.getAttribute('data-salary');
+  const cleanSalary = salaryString.replace(/[^0-9.-]+/g, '');
+
+  return parseInt(cleanSalary, 10);
+}
+
+function sortList() {
+  employees.sort((a, b) => getSalary(b) - getSalary(a));
```

Comment:
The sorting function sorts employees in descending order based on salary. Ensure that this is the intended behavior. If ascending order is needed, switch the subtraction order.